src/basic/mount-util.c

```diff
@@ -543,6 +543,11 @@ int bind_remount_recursive_with_mountinfo(const char *prefix, bool ro, char **bl
                         r = path_is_mount_point(x, NULL, 0);
                         if (IN_SET(r, 0, -ENOENT))
                                 continue;
+                        if (IN_SET(r, -EACCES, -EPERM)) {
+                                /* Even if root user invoke this, FUSE or NFS mount points may not be acceessed. */
```

Comment:
Yeah, we should probably reword the comment. But since @poettering already submitted a follow-up PR, let's first merge this, than that, and then work on any cleanups to avoid conflicts.